Firestore/core/src/remote/grpc_connection.cc

```diff
@@ -119,8 +120,32 @@ class HostConfigMap {
 };
 
 HostConfigMap& Config() {
-  static HostConfigMap config_by_host_;
-  return config_by_host_;
+  static HostConfigMap config_by_host;
+  return config_by_host;
+}
+
+class ClientLanguageHeader {
```

Comment:
I deliberately made this mirror `HostConfigMap` and `Config`. I'm open to suggestions if you think this is overkill.